src/Viewer3D/Viewer3DFacts.h

```diff
@@ -0,0 +1,29 @@
+#ifndef VIEWER3DFACTS_H
+#define VIEWER3DFACTS_H
```

Comment:
This class should used the terminology "Fact". QGC already has the concept of a "Fact"  and a "FactSystem" which is something else. Don't won't to cause confusion.